src/components/SignUp.vue

```diff
@@ -42,6 +42,13 @@ async function signUp() {
     errorMessage.value = (error as Error).message;
     return;
   }
+  try {
+    await createUserOnClimasens();
```

Comment:
just confirming this gets called after the Apigee cookie token gets set?

Reply:
Yes, createUserOnClimasens waits for cookies to be set before sending the request to Climasens.